packages/contracts/contracts/voting/majority/IMajorityVoting.sol

```diff
@@ -180,35 +231,41 @@ interface IMajorityVoting {
     /// @return The participation value.
     function participation(uint256 _proposalId) external view returns (uint256);
 
-    /// @notice Returns all information for a proposal by its ID.
+    /// @notice Returns the vote mode stored in the majority voting settings.
+    /// @return The vote mode parameter.
+    function voteMode() external view returns (VoteMode);
+
+    /// @notice Returns the support threshold parameter stored in the majority voting settings.
+    /// @return The support threshold parameter.
+    function supportThreshold() external view returns (uint64);
+
+    /// @notice Returns the minimum participation parameter stored in the majority voting settings.
+    /// @return The minimum participation parameter.
+    function minParticipation() external view returns (uint64);
+
+    /// @notice Returns the minimum duration parameter stored in the majority voting settings.
+    /// @return The minimum duration parameter.
+    function minDuration() external view returns (uint64);
+
+    /// @notice Returns the minimum voting power required to create a proposa stored in the majority voting settings.
+    /// @return The minimum voting power required to create a proposal.
+    function minProposerVotingPower() external view returns (uint256);
+
+    /// @notice Returns all information for a proposal vote by its ID.
     /// @param _proposalId The ID of the proposal.
     /// @return open Wheter the proposal is open or not.
     /// @return executed Wheter the proposal is executed or not.
-    /// @return startDate The start date of the proposal vote.
-    /// @return endDate The end date of the proposal vote.
-    /// @return snapshotBlock The block number of the snapshot taken for this proposal.
-    /// @return supportThreshold The support threshold in percent.
-    /// @return minParticipation The minimum participation ratio in percent.
-    /// @return totalVotingPower The total number of eligible votes that can be casted.
-    /// @return yes The number of `yes` votes.
-    /// @return no The number of `no` votes.
-    /// @return abstain The number of `abstain` votes.
+    /// @return configuration The configuration of the proposal vote.
+    /// @return tally The current tally of the proposal vote.
     /// @return actions The actions to be executed in the associated DAO after the proposal has passed.
     function getProposal(uint256 _proposalId)
         external
         view
         returns (
             bool open,
             bool executed,
-            uint64 startDate,
-            uint64 endDate,
-            uint64 snapshotBlock,
-            uint64 supportThreshold,
-            uint64 minParticipation,
-            uint256 totalVotingPower,
-            uint256 yes,
-            uint256 no,
-            uint256 abstain,
+            Configuration memory configuration,
+            Tally memory tally,
             IDAO.Action[] memory actions
```

Comment:
Isn't the metadata part of this? I assume we have it via subgraph?

Reply:
Yes, the metadata is only emitted as an event.

Reply:
Can we resolve this?